src/collective/embeddedpage/profiles/default/types/EmbeddedPage.xml

```diff
@@ -35,6 +35,15 @@
   <!-- Enabled behaviors -->
   <property name="behaviors">
     <element value="plone.app.content.interfaces.INameFromTitle" />
+    <element value="plone.app.dexterity.behaviors.discussion.IAllowDiscussion"/>
```

Comment:
@rodfersou did you make sure all those deps are listed as dependencies in setup.py as well?

Reply:
done